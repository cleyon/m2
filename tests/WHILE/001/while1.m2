@sequence X create
@while @X@ < 6
Iteration @X++@
@endwhile
Out of while loop
