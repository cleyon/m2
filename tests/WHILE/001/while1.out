Iteration 0
Iteration 1
Iteration 2
Iteration 3
Iteration 4
Iteration 5
Out of while loop
